gui/services/models.py

```diff
@@ -1343,6 +1343,16 @@ class UPS(Model):
             "a custom command here to perform other tasks before shutdown."
             "default: /sbin/shutdown -p now"),
     )
+    ups_nocommwarntime = models.IntegerField(
+        verbose_name=_('No Communcation Warning Time'),
```

Comment:
Ok please forgive me for this, but s/Communcation/Communication.
OCD + Nitpicking: 1, Suraj: 0 :-P